homeassistant/components/acer_projector/switch.py

```diff
@@ -67,6 +67,13 @@ def setup_platform(
 class AcerSwitch(SwitchEntity):
     """Represents an Acer Projector as a switch."""
 
+    _attr_icon = ICON
+    _attr_extra_state_attributes = {
+        LAMP_HOURS: STATE_UNKNOWN,
+        INPUT_SOURCE: STATE_UNKNOWN,
+        ECO_MODE: STATE_UNKNOWN,
+    }
```

Comment:
This will actually break. As it is a shared mutable across all class instances.

Reply:
This platform supports only one switch, so although this might not be clean it is still OK:
https://github.com/home-assistant/core/blob/8c21a7c7dcc034bc7bc9f4731926f9392f0efa21/homeassistant/components/acer_projector/switch.py#L64

Reply:
It is a platform, so it can be added to your config multiple times to add multiple projectors.
Eventhough a single instance has 1 switch, multiple instances together have multiple switches.